schema/pod.go

```diff
@@ -83,6 +83,15 @@ func (pm *PodManifest) assertValid() error {
 	if pm.ACKind != PodManifestKind {
 		return pmKindError
 	}
+
+	// ensure volumes names are unique (unique key)
+	volNames := make(map[types.ACName]bool, len(pm.Volumes))
```

Comment:
dumb nit, we can save a couple bits if we use `struct{}` in place of `bool`. I'm fine with it either way.